source/Random.cpp

```diff
@@ -34,6 +36,11 @@ namespace {
 #endif
 }
 
+static bool normalBMCached = false;
+static double cachedBMNormal = std::numeric_limits<double>::infinity();
+static const double epsilon = std::numeric_limits<double>::epsilon();
```

Comment:
Those three belong in the anonymous namespace above.

Reply:
Fixed in d6a267ea793b0a066a504e8d069fb4a34793a065.